apps/angular/src/core/services/auth.service.ts

```diff
@@ -0,0 +1,74 @@
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
+import { inject, Injectable } from '@angular/core';
+import { UserSecret } from '@js-camp/core/models/user-secret';
+import { Login } from '@js-camp/core/models/login';
+import { catchError, map, Observable, shareReplay, throwError } from 'rxjs';
+import { UserSecretDto } from '@js-camp/core/dtos/user-secret.dto';
+import { UserSecretMapper } from '@js-camp/core/mappers/user-secret.mapper';
+import { Registration } from '@js-camp/core/models/registration';
+import { ApiErrorMapper } from '@js-camp/core/mappers/api-error.mapper';
+import { ExtendedApiError } from '@js-camp/core/models/api-error';
+
+import { ApiUrlService } from './api-url.service';
+
+/** Authorization service. */
+@Injectable({
+	providedIn: 'root',
+})
+export class AuthService {
+	private readonly http: HttpClient = inject(HttpClient);
```

Comment:
We can omit types for explicit things
```suggestion
	private readonly http = inject(HttpClient);
```